modules/swagger-codegen/src/main/resources/csharp/api.mustache

```diff
@@ -81,10 +81,10 @@ namespace {{packageName}}.Api
         /// <returns></returns>
         public {{classname}}(String basePath)
         {
-            this.Configuration = new Configuration(new ApiClient(basePath));
+            this.Configuration = new {{packageName}}.Client.Configuration(new ApiClient(basePath));
```

Comment:
I believe you should still need to remove `using {{packageName}}.Client;` from line 7 in this file and make all `ApiClient` references fully qualified. I'll pull your branch and the examples in Windows.